### PR TITLE
fixed bug of WebSocket that it can not receive over 4096 byte

### PR DESCRIPTION
**Pull request by dualface**
see #3981

### CODE REVIEW

**Comment by dualface** [comment]
see #4014 

Thanks edwardzhou and HoGarfield.

File: extensions/network/WebSocket.h [hunk 1]
```diff
@@ -58,9 +58,10 @@ class WebSocket
      */
     struct Data
     {
-        Data():bytes(NULL), len(0), isBinary(false){}
+        Data():bytes(NULL), len(0), issued(0), isBinary(false){}
         char* bytes;
-        int len;
+        size_t len;
+        int issued;
```
Comment (dumganhar) [inline extensions/network/WebSocket.h:64]: Typo? `issued` --> `isUsed`?

Comment (edwardzhou) [inline extensions/network/WebSocket.h:64]: issued (issue + ed) means the bytes was sent

Comment (dumganhar) [inline extensions/network/WebSocket.h:64]: Got it. @edwardzhou , thanks

Comment (dumganhar) [inline extensions/network/WebSocket.h:64]: @edwardzhou , did you test sending more than 4096 bytes to websocket server?
@dualface has already rechecked it, and he said it also has problem to send messages more than 4096bytes

Comment (edwardzhou) [inline extensions/network/WebSocket.h:64]: exactly. I will setup a temporary websocket server for this case for you today.

Comment (dumganhar) [inline extensions/network/WebSocket.h:64]: That will be great, thanks in advance. :)

Comment (edwardzhou) [inline extensions/network/WebSocket.h:64]: I have submit another pull request for this, since I deleted own original repo before.

**Comment by dumganhar** [comment]
Merged into develop branch here https://github.com/cocos2d/cocos2d-x/pull/5008 .
I closed this PR since this is not a critical bug. :)
